apps/frontend/pages/404.js

```diff
@@ -0,0 +1,35 @@
+import link from 'next/link'
+import styles from '../styles/404.module.css'
+
+export default function Custom404() {
```

Comment:
please use `.tsx` instead of `.js`

Reply:
in docs they used .js so i was a bit confused. I created this file with a lot help of next docs